pkgs/top-level/all-packages.nix

```diff
@@ -11989,7 +11997,7 @@ with pkgs;
 
   busybox = callPackage ../os-specific/linux/busybox {
     # Fixes libunwind from being dynamically linked to a static binary.
-    stdenv = if (stdenv.targetPlatform.useLLVM or false) then
+    stdenv = if stdenv.cc.isClang && stdenv.hostPlatform.unwinderlib == "libunwind" then
       overrideCC stdenv buildPackages.llvmPackages.clangNoLibcxx
     else stdenv;
   };
```

Comment:
Wait, what? This seems like a toolchain problem rather than something we should hack around here. Why would `pkgsStatic.busybox` link against a dynamic `libunwind`?

Reply:
`pkgsStatic` isn't the problem, `pkgsLLVM` is the problem. `busybox` is static and `libunwind` is dynamic.